src/nbl/video/COpenGLFeatureMap.h

```diff
@@ -1056,6 +1062,8 @@ class COpenGLFeatureMap
 		NBL_OES_copy_image,
 		NBL_OES_viewport_array,
 		NBL_OES_draw_elements_base_vertex,
+		NBL_OES_geometry_shader,
+		NBL_OES_tessellation_shader,
 
 		NBL_OpenGL_Feature_Count
 	};
```

Comment:
Yep, I'll deduplicate them soon

Reply:
https://github.com/Devsh-Graphics-Programming/Nabla/blob/d835cb5cb17413dad21c04cd8836c0d7716b3457/src/nbl/video/SOpenGLContextLocalCache.cpp#L130-L137

I want to somehow get these from PhysDev Instead of glFeatures so I could remove them, they are duplicate of maxDescriptor stuff

Reply:
ugh this shit gets tricky because I need to revisit what they're being used for cause they're badly named

Reply:
> ugh this shit gets tricky because I need to revisit what they're being used for cause they're badly named

Ok I removed every other useless var from glFeatureMap that can be queried from physicalDevice.
the only remaining ones are the ones used `SOpenGLContextLocalCache` I mentioned above which I cannot remove because I don't have access to the physical device there